src/useragentparser/DefaultUserAgentParser.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { UAParser } from 'ua-parser-js';
```

Comment:
What is the difference of using this vs BrowserBehavior?

Reply:
ua-parser-js is better at getting more specific information that detect-browser used in BrowserBehavior. We started using ua-parser-js when we released meeting events. I separated it so that it can be more generally used within SDK.

Reply:
Didn't know that. Can you document it here then? Maybe we should add an action item to migrate browser behavior to ua-parser-js in v3?

Reply:
Yeah sounds good. Added a line over this import and will track the v3 one. Thanks!